.dockerignore

```diff
@@ -0,0 +1,29 @@
+*.pyc
+include/
+lib/
+staticfiles/
+media/
+.build
+
+# sqlite3 db
+*.db
+*.sqlite3
+
+# sublime stuff
+*.sublime-workspace
+*.sublime-project
```

Comment:
Ah, that's annoying. Leave it here then, I guess, but maybe group them at the bottom under a comment?